lib/core/utils.js

```diff
@@ -1486,33 +1486,18 @@ RESUtils.niceDateTime = function(d, usformat) {
 	return `${dateString} ${hours}:${minutes}:${seconds}`;
 };
 RESUtils.niceDateDiff = function(origdate, newdate) {
-	// Enter the month, day, and year below you want to use as
-	// the starting point for the date calculation
 	if (!newdate) {
 		newdate = new Date();
```

Comment:
Might as well make this a default param while you're at it, I guess.
`newdate` is always a Date object or undefined, so no usages should have to be changed.